pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotLeadControllerRestletResource.java

```diff
@@ -83,8 +92,8 @@ public LeadControllerResponse getLeadersForAllTables() {
       String partitionName = LeadControllerUtils.generatePartitionName(partitionId);
       String participantInstanceId =
           getParticipantInstanceIdFromExternalView(leadControllerResourceExternalView, partitionName);
-      leadControllerEntryMap
-          .putIfAbsent(partitionName, new LeadControllerEntry(participantInstanceId, new ArrayList<>()));
+      leadControllerEntryMap.putIfAbsent(partitionName,
+          new LeadControllerEntry(participantInstanceId, new ArrayList<>()));
```

Comment:
seems like an unrelated formatting change. I'd revert that

Reply:
Seems my IDE's style format is never set correct. Need to double check that again.